spotlight-client/src/charts/utils.ts

```diff
@@ -23,6 +23,11 @@ import { isItemToHighlight, ItemToHighlight } from "./types";
 
 const FADE_AMOUNT = 0.45;
 
+export function generateHatchFill(id: string): string {
+  const cleanId = id.replace(/[^\w\d]/g, "");
+  return `url(#${cleanId})`;
```

Comment:
I'm new to this, what is this `url(#...)`?

Reply:
It's pointing to defs with pattern by `id`